server/telemetry.go

```diff
@@ -8,15 +8,35 @@ import (
 	"net/http"
 
 	"github.com/mattermost/mattermost-plugin-calls/server/telemetry"
+
+	"github.com/mattermost/mattermost/server/public/model"
+
+	"github.com/rudderlabs/analytics-go"
 )
 
 const (
 	// server-side events
-	evCallStarted     = "call_started"
-	evCallEnded       = "call_ended"
-	evCallUserJoined  = "call_user_joined"
-	evCallUserLeft    = "call_user_left"
-	evCallNotifyAdmin = "call_notify_admin"
+	evCallStarted                   = "call_started"
+	evCallEnded                     = "call_ended"
+	evCallUserJoined                = "call_user_joined"
+	evCallUserLeft                  = "call_user_left"
+	evCallNotifyAdmin               = "call_notify_admin"
+	evCallConcurrentSessionsWarning = "call_concurrent_sessions_warning"
+)
+
+var (
+	enterpriseSKUs = []string{model.LicenseShortSkuEnterprise}
+	// currently unused
+	// professionalSKUs = []string{model.LicenseShortSkuProfessional, model.LicenseShortSkuEnterprise}
+
+	// We only need to map events that require a SKU (i.e., licensed features). Anything available on unlicensed
+	// servers will map to null as expected.
+	eventToSkusMap = map[string][]string{
+		"user_start_recording": enterpriseSKUs,
+		"user_stop_recording":  enterpriseSKUs,
+		"live_captions_on":     enterpriseSKUs,
+		"live_captions_off":    enterpriseSKUs,
+	}
```

Comment:
telemetry PR?

Reply:
Likely got it as the parent PR got merged. Will rebase.